File: lab1/src/main/java/Service/DestinationService.java
package Service;

import Domain.Destination;
import Gateway.DestinationGateway;
import Validator.DestinationValidator;

public class DestinationService extends BaseService<Destination, DestinationGateway> {
  public DestinationService() {
    super(new DestinationGateway(), new DestinationValidator());
  }
}

File: proiect_c# V2/Services/Service/DestinationService.cs
using csharp.Domain;

namespace csharp.Services.Service
{
	public class DestinationService : BaseService<Destination, IDestinationRepository>
	{
		public DestinationService(IDestinationRepository repository, DestinationValidator validator) : base(repository, validator)
		{
		}
	}
}